base_bank/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Author: Andrius Laukavičius. Copyright: JSC Focusate.
+# See LICENSE file for full copyright and licensing details.
+{
+    'name': "Banks Improvements",
+    'version': '14.0.2.0.0-rc.1',
+    'summary': 'bank, bank accounts, improve, sequence',
+    'license': 'LGPL-3',
+    'author': "Focusate",
+    'website': "www.focusate.eu",
+    'category': 'Hidden/Tools',
+    'depends': [
+        # odoo
+        'base'
+    ],
+    'data': [
+        'views/res_partner_bank_views.xml'
+    ],
+    'installable': True,
```

Comment:
Is this a whole new module? We already had 'partner_bank_extended', so maybe those could be merged.

Reply:
I guess you are right, its useless as separate one (forgot about that one). Will merge with it later.